libpod/adapter/runtime_remote.go

```diff
@@ -246,3 +271,44 @@ func (ci *ContainerImage) History(ctx context.Context) ([]*image.History, error)
 	}
 	return imageHistories, nil
 }
+
+// LookupContainer ...
+func (r *RemoteRuntime) LookupContainer(idOrName string) (*Container, error) {
+	container, err := iopodman.GetContainer().Call(r.Conn, idOrName)
+	if err != nil {
+		return nil, err
+	}
+
+	ctr, err := listContainerDataToContainer(container)
+	if err != nil {
+		return nil, err
+	}
+	return ctr, nil
+}
+
+// listContainerDataToContainer takes a varlink listcontainerData struct and makes
+// an "adapted" Container
+func listContainerDataToContainer(listData iopodman.ListContainerData) (*Container, error) {
+	created, err := splitStringDate(listData.Createdat)
+	if err != nil {
+		return nil, err
+	}
+	rc := remoteContainer{
+		ID:         listData.Id,
+		Image:      listData.Image,
+		ImageID:    listData.Imageid,
+		Command:    listData.Command,
+		Created:    created,
+		RunningFor: listData.Runningfor,
+		Status:     listData.Status,
+		//ports: //map[ocicni.portmapping]
+		RootFsSize: listData.Rootfssize,
+		RWSize:     listData.Rwsize,
+		Names:      listData.Names,
+		//Labels:
```

Comment:
Why are these commented out, either implement or remove.

Reply:
already responded to this. added a todo comment for clarification.